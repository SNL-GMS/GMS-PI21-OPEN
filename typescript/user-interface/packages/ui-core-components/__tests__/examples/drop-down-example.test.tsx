import { render } from '@testing-library/react';
import React from 'react';

import { DropDownExample } from '../../src/ts/examples/drop-down-example';

// set up window alert and open so we don't see errors
(window as any).alert = jest.fn();
(window as any).open = jest.fn();

it('renders a component', () => {
  const { container } = render(<DropDownExample />);
  expect(container).toMatchSnapshot();
});
