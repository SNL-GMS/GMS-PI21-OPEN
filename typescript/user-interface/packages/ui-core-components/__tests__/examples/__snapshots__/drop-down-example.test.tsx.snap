// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a component 1`] = `
<div>
  <div
    class="ag-dark"
    style="flex: 1 1 auto; position: relative; width: 700px;"
  >
    <div
      class="dropdown-container"
    >
      <span
        class="dropdown-selector"
        data-cy="-dropdown"
      >
        <div
          class="bp4-html-select"
        >
          <select
            title="undefined"
          >
            <option
              value="Example Value One"
            >
              Example Value One
            </option>
            <option
              value="Example Value Two"
            >
              Example Value Two
            </option>
            <option
              value="Example Value Three"
            >
              Example Value Three
            </option>
          </select>
          <span
            class="bp4-icon bp4-icon-double-caret-vertical"
            icon="double-caret-vertical"
          >
            <svg
              data-icon="double-caret-vertical"
              height="16"
              viewBox="0 0 16 16"
              width="16"
            >
              <desc>
                Open dropdown
              </desc>
              <path
                d="M5 7h6a1.003 1.003 0 00.71-1.71l-3-3C8.53 2.11 8.28 2 8 2s-.53.11-.71.29l-3 3A1.003 1.003 0 005 7zm6 2H5a1.003 1.003 0 00-.71 1.71l3 3c.18.18.43.29.71.29s.53-.11.71-.29l3-3A1.003 1.003 0 0011 9z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
        </div>
      </span>
    </div>
    <div>
      Selected: Example Value One
    </div>
  </div>
</div>
`;
